spec/ruby/core/array/shuffle_spec.rb

```diff
@@ -58,9 +58,9 @@
 
   it "raises a RangeError if the value is less than zero" do
     value = mock("array_shuffle_random_value")
-    value.should_receive(:to_int).and_return(-1)
+    value.should_receive(:to_int).at_least(1).times.and_return(-1)
     random = mock("array_shuffle_random")
-    random.should_receive(:rand).and_return(value)
+    random.should_receive(:rand).at_least(1).times.and_return(value)
```

Comment:
> I modified the array/shuffle_spec to "at least once" behavior which is consistent with all the tests around it.
> This way the spec passes before and after this change.

Why does this fail now and not before, and why does it not fail on CRuby?
I think this spec shouldn't need to change.